CONTRIBUTION.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+Some General rules regarding contributing ethics.
+
+# Installation and Setup
+
+To setup the project, refer [here](https://github.com/iit-technology-ambit/common-backend/blob/master/README.md)
+
+# Ettiquites
+
+We follow certain etiquettes which include docstrings, annotation, import sorting etc.
+
+# Tests
+
+The project is tested via Travis CI. The following tests are performed automatically:
+
+- mypy: For type checking
+- pylama: For code auditing
```

Comment:
Please mention what all we use in pylama.

Also, please remove .mypy_cache.

Reply:
Sorry, this is another branch, that's why its still there